### DISCUSSION TITLE
deploy generic ui

### DISCUSSION THREAD

**Discussion by bluebrown** [swagger-api/swagger-ui#8051] [Q&A]
Hi I am wondering if its possible to deploy the ui only once and request from the webpage the swagger.json or similar from multiple backends. So that I can avoid having each backend service its own ui.

**Comment by mathis-m** [comment]
Depends on what backend library you use. But you have always the option to privide a UI separately. Just like you can use the hosted version at petstore3.swagger.io for any OAS spec you like.

To make it integrate nicely you probably need to have some selector to select the api you want to inspect.
But this can be archived via the plugin api.
